gensrc/thrift/StatusCode.thrift

```diff
@@ -112,5 +112,8 @@ enum TStatusCode {
     CAPACITY_LIMIT_EXCEED = 58,
 
     SHUTDOWN = 59, // the service is shutting down
+
+    BIG_QUERY_CPU_SECOND_LIMIT_EXCEEDED = 60,
+    BIG_QUERY_SCAN_ROWS_LIMIT_EXCEEDED = 61,
```

Comment:
I'm a little confused about these items name. 
So do we have a concept of "BIG_QUERY"? Which kind of queries can be treated as BIG_QUERY?
And what do these two items mean?

Reply:
[Resource group | StarRocks](https://docs.starrocks.io/docs/administration/management/resource_management/resource_group/)
https://docs.starrocks.io/docs/administration/management/resource_management/resource_group/

In resource group,  we have definition of "big query" by setting some limits: 

> On the basis of the above resource consumption restrictions, you can further restrict the resource consumption for big queries with the following parameters:
> 
> - big_query_cpu_second_limit: This parameter specifies the CPU upper time limit for a big query on a single BE. Concurrent queries add up the time. The unit is second. This parameter takes effect only when it is set greater than 0. Default: 0.
> - big_query_scan_rows_limit: This parameter specifies the scan row count upper limit for a big query on a single BE. This parameter takes effect only when it is set greater than 0. Default: 0.
> - big_query_mem_limit: This parameter specifies the memory usage upper limit for a big query on a single BE. The unit is byte. This parameter takes effect only when it is set greater than 0. Default: 0.
>

Reply:
thanks for letting me know it.
Then I have no more questions.